src/Microsoft.DocAsCode.Build.Engine/CompilePhaseHandlerWithIncremental.cs

```diff
@@ -88,6 +89,7 @@ private void PostHandle(List<HostService> hostServices)
                 h.SaveIntermediateModel(IncrementalContext);
             }
             IncrementalContext.UpdateBuildVersionInfoPerDependencyGraph();
+            SaveContextInfo(hostServices);
```

Comment:
i'd like to allow plugin writer to decide which phase to load/save the context info. but since it might be confusing , remove it.